timechart/utilities/utils.py

```diff
@@ -4,21 +4,45 @@
 from qtpy.QtGui import QColor
 from qtpy.QtWidgets import QMessageBox
 
+
+global STRIPTOOL_PREDEFINED_COLORS
```

Comment:
If TimeChart is importing a StripTool config file, which contains a list of preferred colors, TimeChart will populate STRIPTOOL_PREDEFINED_COLORS. When TimeChart picks a random color for a curve, it'll pick from STRIPTOOL_PREDEFINED_COLORS instead of the predefined colors of TimeChart's own.

On the other hand, if TimeChart imports its own JSON config file, it'll pick a random color for a curve from its own color list.

Reply:
Docs added.